book/src/bindings/README.md

```diff
@@ -0,0 +1,9 @@
+# Bindings
+
+PRQL has bindings for many languages. These include:
+
+- Java
+- Javascript
+- Python
+- R
+- Rust
```

Comment:
I agree the existing "grayed out" parts were not good, so thanks a lot for adding these in.

What do you think about copying the part from `SUMMARY.md` so we also get the links? The links would have be adjusted because this page is a level down, but otherwise it's a copy & paste; and it makes it obvious how to keep it up to date.

BTW it's a shame there isn't a way of doing that automatically, for these standard pages...

Reply:
Good thought to snag the items from `SUMMARY.md`